### PR TITLE
Add playtest-20221222 announcement page

### PR DESCRIPTION
**Pull request by PunkPun**

### CODE REVIEW

File: news/_posts/2022-12-22-playtest-20221222.md [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+Just in time for the holiday season, we are releasing [Playtest 20221222]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
+* Fixed a few Red Alert missions
+* Fixed carryalls in rare cases destrying their cargo
+* Added caryalls to the D2K spectator economy statistics
+* Added new maps to the D2K map pool
+* A few other minor fixes
+
+We expect this will be our final playtest before a full release on New Year's, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or [GitHub](https://github.com/OpenRA/OpenRA/) issues.
```
Comment (abcdefg30) [inline news/_posts/2022-12-22-playtest-20221222.md:22]:
```suggestion
We expect this will be our final playtest before a full release on New Year's, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```

Comment (PunkPun) [reply news/_posts/2022-12-22-playtest-20221222.md:22]: sure

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
+* Fixed a few Red Alert missions
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:16]:
```suggestion
* Fixed the winning condition in a few Red Alert missions
```

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* Fixed carryalls in rare cases destrying their cargo
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:17]:
```suggestion
* Fixed carryalls in rare cases destroying their cargo
```

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+* Added caryalls to the D2K spectator economy statistics
+* Added new maps to the D2K map pool
+* A few other minor fixes
+
+We expect this will be our final playtest before a full release on New Year's, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:22]: New Year's *Day*? So first of January. Or New Year's Eve? I wouldn't mention this at all but just write
```suggestion
We expect this will be our final playtest before a full release, so we highly encourage everybody to play some games and missions and let us know if you find any issues in the comments below, on [our forum](https://forum.openra.net), [Community Discord](https://discord.openra.net), or on [GitHub](https://github.com/OpenRA/OpenRA/issues).
```

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 5]
```diff
@@ -0,0 +1,113 @@
...
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
+* Fixed a few Red Alert missions
+* Fixed carryalls in rare cases destrying their cargo
+* Added caryalls to the D2K spectator economy statistics
+* Added new maps to the D2K map pool
+* A few other minor fixes
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:20]: This feels a bit off. Maybe we can reword it to something along the lines of `Further fixes to the map editor and core engine`?

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Fixed a few pathfinder crashes
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:15]:
```suggestion
* Fixed several pathfinder-related crashes
```
? (To not have "Fixed a few" repeated twice in a row, unless that is intentional.)

**Review by abcdefg30 — COMMENTED** [review]
Note: We also need to adjust the tag in `_config.yml`.

Preview:
grafik

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 7]
```diff
@@ -0,0 +1,113 @@
+---
+kind: article
+title: "Playtest 20221223"
+author: "PunkPun"
+created_at: 2022-12-22 23:00 +0000
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:5]: This will also want to be updated I guess.

File: news/_posts/2022-12-23-playtest-20221223.md (hunk 5 above)
Comment (PunkPun) [reply news/_posts/2022-12-23-playtest-20221223.md:20]: Changed it to `Other minor fixes`

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 8]
```diff
@@ -0,0 +1,115 @@
...
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
+
+* Dropped support for macOS 10.9 and 10.10
+* Updated macOS packaging, linux and macOS dependencies
```
Comment (pchote) [inline news/_posts/2022-12-23-playtest-20221223.md:16]: * Added macOS universal binaries for Apple Silicon

and put this before the dropped support line.

Comment (PunkPun) [reply news/_posts/2022-12-23-playtest-20221223.md:16]: ok

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 9]
```diff
@@ -0,0 +1,116 @@
...
+author: "PunkPun"
+created_at: 2022-12-22 23:00 +0000
+disqus_id: "playtest-20221223"
+permalink: "/news/playtest-20221223/"
+---
+
+<div style="border-radius: 4px; background-color: #272d2c; padding: 5px">
+<div style="margin: -10px 5px" markdown="1">
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog):
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:13]:
```suggestion
Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog/e5fda938ada5be95883b9631e2dd1a3609fead45):
```

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 10]
```diff
@@ -0,0 +1,116 @@
...
+
+Just in time for the holiday season, we are releasing [Playtest 20221223]({{ '/download' | relative_url }}) with [several important fixes and minor features](https://github.com/OpenRA/OpenRA/wiki/Changelog/e5fda938ada5be95883b9631e2dd1a3609fead45):
+
+* Added macOS universal binaries for Apple Silicon
+* Dropped support for macOS 10.9 and 10.10
+* Updated macOS packaging, linux and macOS dependencies
+* Fixed several pathfinder-related crashes
+* Fixed the victory condition in a few of the Red Alert missions
+* Fixed carryalls in rare cases destroying their cargo
+* Added caryalls to the D2K spectator economy statistics
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:21]:
```suggestion
* Added caryalls to the Dune 2000 spectator economy statistics
```

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 11]
```diff
@@ -0,0 +1,116 @@
...
+* Added new maps to the D2K map pool
```
Comment (abcdefg30) [inline news/_posts/2022-12-23-playtest-20221223.md:22]:
```suggestion
* Added new maps to the Dune 2000 map pool
```

File: news/_posts/2022-12-23-playtest-20221223.md (hunk 10 above)
Comment (PunkPun) [reply news/_posts/2022-12-23-playtest-20221223.md:21]: Feels weird but formal style I guess

**Review by abcdefg30 — APPROVED** [review]

File: news/_posts/2022-12-23-playtest-20221223.md [hunk 12]
```diff
@@ -0,0 +1,116 @@
+---
+kind: article
+title: "Playtest 20221223"
+author: "PunkPun"
+created_at: 2022-12-23 18:00 +0000
+disqus_id: "playtest-20221223"
```
Comment (pchote) [inline news/_posts/2022-12-23-playtest-20221223.md:6]: We would generally keep the previous comments for posts that are tacking extra info on top of a previous one.

Comment (PunkPun) [reply news/_posts/2022-12-23-playtest-20221223.md:6]: makes sense
